pythran/tables.py

```diff
@@ -873,6 +873,14 @@ def update_effects(self, node):
     logger.warn("Pythran support disabled for module: omp")
     del MODULES["omp"]
 
+# check and deletion of unimplemented numpy methods
+unsupported_numpy_methods = []
+for method in MODULES['numpy']:
+    if not method in sys.modules['numpy'].__dict__ and not (method[-1:] == '_' and method[:-1] in cxx_keywords and method[:-1] in sys.modules['numpy'].__dict__):
+        unsupported_numpy_methods.append(method)
+for method in unsupported_numpy_methods:
+    del MODULES['numpy'][method]
```

Comment:
why a double loop here? just iterator over ``MODULES['numpy'].keys()`` !

BTW, your code is certainly not pep8 compliant, please run the validation before submitting a PR ;-)